OrbitQt/TopDownWidget.cpp

```diff
@@ -28,7 +28,30 @@ void TopDownWidget::SetTopDownView(std::unique_ptr<TopDownView> top_down_view) {
   ui_->topDownTreeView->sortByColumn(TopDownViewItemModel::kInclusive, Qt::DescendingOrder);
   ui_->topDownTreeView->header()->resizeSections(QHeaderView::ResizeToContents);
 
-  on_searchLineEdit_textEdited(ui_->searchLineEdit->text());
+  onSearchLineEditTextEdited(ui_->searchLineEdit->text());
+}
+
+static void CopyFromIndices(OrbitApp* app, const QModelIndexList& indices) {
```

Comment:
I think this function should not depend on OrbitApp. Ideally it returns a std::string. So it can be tested independently. I would move the `app->SetClipboard`-call into the key-press handler or have a separate function.